chromium/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "chromium",
+  "version": "1.0.0",
+  "description": "",
+  "main": "background.js",
+  "dependencies": {
+    "chai": "^4.1.2",
+    "mocha": "^4.0.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "mocha"
```

Comment:
"test": "eslint . && mocha"

Reply:
eslint tests are run separately by travis

Reply:
No, I mean move it here.

Reply:
This is the main entry point for unit testing. So I don't think it should also run the linter.

This is a problem if, for example, I want to use `npm test` to run the unit tests on travis. It would require that eslint be installed, and it would run the linter twice. So if there is a lint failure both the travis linting job, and the unit test job would fail, regardless of whether the unit tests actually passed.

Reply:
Remove the old ESLint then.

Reply:
Also you can make two scripts, `lint` and `unittest` and run both from `test` script, allowing to test either ESLint, unit tests or both.

Reply:
linting is not specific to the either the node package context, nor the extension context. So I don't think it belongs here, in a node specific context.

Reply:
@cowlicks It's specific to Node context. It can only be done with Node.js.